docs/src/content/docs/guides/connectionoptions.mdx

```diff
@@ -54,28 +49,77 @@ Creating an Ultravox call that works with Twilio is just like creating a WebRTC
     </tr>
 </table>
 
-Adding these to the request body when creating the call would look like this:
-
+Example request body for an outgoing phone call:
 ```javascript
 {
   "systemPrompt": "You are a helpful assistant...",
   ...
   "medium": {
-    "twilio": {}
+    "telnyx": {} // or "twilio": {} or "plivo": {}
   },
   "firstSpeaker": "FIRST_SPEAKER_USER"
 }
 ```
 
-Ultravox will return a `joinUrl` that can then be used with Twilio for outgoing or incoming calls.
+### Provider-Specific Integration
 
-### Outgoing Calls
+<Tabs>
+<TabItem label="Telnyx">
 
-It only takes two steps to make an outgoing call to regular phone numbers through Twilio:
+#### Outgoing Calls with Telnyx
 <Steps>
-1. **Create an Ultravox Call** → Create a new call (see [above](#creating-a-phone-call-with-twilio)), and get a `joinUrl`.
+1. **Create an Ultravox Call** → Create a new call as shown above with `medium: { "telnyx": {} }`, `firstSpeaker: "FIRST_SPEAKER_USER"`, and get a `joinUrl`.
 
-1. **Initiate Twilio Phone call** → Use the `joinUrl` with a Twilio [`<Stream>`](https://www.twilio.com/docs/voice/twiml/stream).
+2. **Initiate Telnyx Phone call** → Use the `joinUrl` with a TeXML `<Stream>`:
+
+    ```javascript
+    // Example using the telnyx node library
+    const call = await telnyx.calls.create({
+      connection_id: "uuid",
+      to: phoneNumber,
+      from: telnyxPhoneNumber,
+      stream_url: joinUrl,
+      stream_track: "both_tracks"
+    });
+    ```
+
+    Or using TeXML:
+    ```xml
+    <?xml version="1.0" encoding="UTF-8"?>
+    <Response>
+      <Start>
+        <Stream url="${joinUrl}" track="both_tracks" />
+      </Start>
```

Comment:
I think this should be `<Connect><Stream url=${your-join-url} /></Connect>` but I still can't test this.  Given the issues with my account, maybe we can have Zach invite you instead and you can test it out.

The JS above seems reasonable, but I don't have a way to verify that either.

Reply:
ok making this change for now. will fix up if needed once we can test fully.